apps/actions/templates/index.jade

```diff
@@ -0,0 +1,13 @@
+extends ../../../components/layout/index
+
+block meta
+  != apollo.styles
+
+block body
+  #apolloMount
+    != apollo.html
+
+block scripts
+  script( src= asset('/assets/actions.js') )
+  script.
+    window.__APOLLO_STATE__ = !{JSON.stringify(apollo.state).replace(/</g, '\\u003c')};
```

Comment:
I need to write a JSX layout.

Reply:
Though I suppose we'd need a new search/user bar to do that.

Reply:
Yeah true.